docs/digger/installation/run-installer.asciidoc

```diff
@@ -0,0 +1,57 @@
+== Run the Digger Installer
+
+To run the Digger installer, you should:
+
+. Step 1
++
+Clone the https://github.com/aerogear/digger-installer[Digger Installer repo]
+
+. Step 2
++
+Update the `inventory-sample` file in the repo to provide the information about your cluster.
+
+. Step 3
++
+Update the configurations of various tasks. You should check the following configurations:
++
+* Check the variables defined in `vars/buildfarm.yml` file and change them if required. 
```

Comment:
We should avoid pointing the user to the var file. The variables here are intended to be defaults normally. It is better to suggest to the user that they can override certain variables by passing them at the command line.